geniza/corpus/views.py

```diff
@@ -15,3 +23,70 @@ def get_queryset(self, *args, **kwargs):
         """Don't show document if it isn't public"""
         queryset = super().get_queryset(*args, **kwargs)
         return queryset.filter(status=Document.PUBLIC)
+
+
+# --------------- Publish CSV to sync with old PGP site --------------------- #
+
+
+DocumentRow = namedtuple(
+    "DocumentRow",
+    [
+        "pgpid",
+        "library",
+        "shelfmark",
+        "shelfmark_alt",
+        "rectoverso",
+        "type",
+        "tags",
+        "joins",
+        "descr",
+        "editor",
+    ],
+)
+
+
+def tabulate_queryset(queryset):
+    reverse_recto_verso_lookup = {
+        code: label.lower() for code, label in TextBlock.RECTO_VERSO_CHOICES
+    }
+
+    for doc in queryset:
+        all_fragments = doc.fragments.all()
+        rectoverso_q = doc.textblock_set.filter(~Q(side="")).first()
+
+        row = DocumentRow(
```

Comment:
I don't remember why you guys decided to the named tuple for the admin export, but I don't want to devote that much space to this export! Let's just return (yield) a list of values for each row. You can add comments for the labels (where/if not obvious) so we don't lose track.

Reply:
Yeah, I can certainly yield a list instead of the dict, but we'll need to then list out the headers anyway in the `export_to_csv_response`, where we currently use `DocumentRow._fields`. I can do that, but I don't think it'll save that much space. Would you mind confirming?

Reply:
yes, please just provide a list of fields and generate a list of values

Reply:
✅  c4b7918